cypress/support/step_definitions/validations.js

```diff
@@ -298,27 +298,25 @@ Given(
     };
 
     cy.sendMessagetoPlatforms(requestMap).then((result) => {
-      fireLog.info('Performance validation has stopped').then(() => {
-        cy.wrap(result)
-          .then((response) => {
-            if (response && Array.isArray(response)) {
-              response.map((res) => {
-                cy.log(JSON.stringify(res));
-              });
-            }
-          })
-          .then(() => {
-            if (result.error) {
-              cy.log('Failed to fetch and validate the performance metrics').then(() => {
-                assert(false, result.error);
-              });
-            } else {
-              result.map((response) => {
-                assert.equal(true, response?.success, response?.message);
-              });
-            }
-          });
-      });
+      cy.wrap(result)
+        .then((response) => {
+          if (response && Array.isArray(response)) {
+            response.map((res) => {
+              fireLog.info(JSON.stringify(res));
+            });
+          }
+        })
+        .then(() => {
+          if (result.error) {
+            fireLog.info('Failed to fetch and validate the performance metrics').then(() => {
```

Comment:
I think Joe created a fireLog command that can be used that handles the `fireLog.info() -> fireLog.assert()` sequence that you have here. Please check it out: https://github.com/rdkcentral/firebolt-certification-suite/blob/dev/cypress/support/cypress-support/src/utils.js#L882 and use it if it handles this case.

Reply:
I tried using firelog.error, but didn't see any difference in the logs, so I kept it as is. Here are the reports for [firelog.assert]( https://reporting.firecert.comcast.com/cucumberReports/bd10f4d3-e219-4cc3-a4cc-c8b0c9ef15f4/index.html
) and [firelog.error](https://reporting.firecert.comcast.com/cucumberReports/a5644971-494d-4579-bf66-d909bf767823/index.html)